package com.byazt.glq;

import com.byazt.ete.ic;
import com.byazt.ete.to;
import com.bykv.vk.component.ttvideo.player.MediaPlayer;

/* JADX INFO: loaded from: classes.dex */
@com.byazt.zqa.c(c = {0, 1, MediaPlayer.MEDIA_PLAYER_OPTION_ENABLE_NOTIFY_SEI_IMMEDIATELY_BEFORE_FIRSTFRAME, 13})
public class tt {
    public static int c(ic icVar) {
        if (to.uj(icVar)) {
            return ((4 == to.tt(icVar)) && (1 == com.byazt.apd.tt.c().n())) ? 1 : -1;
        }
        return -1;
    }
}
